app/models/figure.rb

```diff
@@ -9,5 +10,12 @@ class Figure < ActiveRecord::Base
   validates :era, presence: true
   validates :claim_to_fame, presence: true
 
-
+  def average_rating
+    ratings = Rating.where(figure_id: id)
```

Comment:
I wonder if this line is necessary, do you get the same thing if you do `self.ratings` or something of that nature?